### ISSUE TITLE
Document what is necessary to run addons/dev-helpers/bin/generator-radius-attributes.sh

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#6559]
**Describe the bug**
During the release procedure, we must run addons/dev-helpers/bin/generator-radius-attributes.sh

The instructions of the release procedure must be improved to include what needs to be installed in order to run this script

Here is the output I got:
```
# addons/dev-helpers/bin/generator-radius-attributes.sh
addons/dev-helpers/bin/generator-radius-attributes.sh: line 5: json_xs: command not found
addons/dev-helpers/bin/generator-radius-attributes.go:6:2: no required module provides package github.com/inverse-inc/go-radius/dictionary: go.mod file not found in current directory or any parent directory; see 'go help modules'
addons/dev-helpers/bin/generator-radius-attributes.sh: line 9: perltidy: command not found
addons/dev-helpers/bin/generator-radius-attributes.sh: line 7: tpage: command not found
```

Instructions are here: https://wiki.inverse.ca/focus/PacketFence/ReleaseProcedure

**Comment by jrouzierinverse** [comment]
The problem is with packaging our own perl libraries.
I need to adjust some ENV variables.

**Comment by jrouzierinverse** [comment]
This is fixed by 9cfa3a5d9a96989d1f8eabb2ba232083d05ab7c0
